akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolConductor.scala

```diff
@@ -67,10 +72,15 @@ private object PoolConductor {
       val doubler = Flow[SwitchCommand].mapConcat(x ⇒ x :: x :: Nil) // work-around for https://github.com/akka/akka/issues/17004
       val route = b.add(new Route(slotCount))
       val retrySplit = b.add(new RetrySplit())
+      val flatten = Flow[RawSlotEvent].mapAsyncUnordered(slotCount) {
```

Comment:
Unordered is fine. We don't want a slow response on one connection block us from seeing an update from another connection.